packages/ninjamsrv/PKGBUILD

```diff
@@ -0,0 +1,44 @@
+# Maintainer: OSAMC <https://github.com/osam-cologne/archlinux-proaudio>
+# Contributor: jakob <grandchild@gmx.net>
+# Contributor: Florian Hülsmann <fh@cbix.de>
+
+_name=ninjam
+pkgname=ninjamsrv
+pkgver=0.080
+pkgrel=1
+pkgdesc='Ninjam online jamming server'
+arch=(x86_64 aarch64)
+url='https://cockos.com/ninjam/'
+license=(GPL2)
+depends=(gcc-libs)
+makedepends=()
+optdepends=()
+groups=(pro-audio)
+conflicts=(ninjam-server-git)
```

Comment:
The correct thing would be for the `ninjam-server-git` package to have `provides=(ninjam-server)` and this one too. We can't change `ninjam-server-git`, but maybe we should add  `provides=(ninjam-server)`.

Reply:
Only using `provides` will lead to a conflict during installation, while explicit `conflicts` warns the user upon installation and asks about removing the conflicting package. See [the wiki](https://wiki.archlinux.org/title/PKGBUILD#provides):
> Packages providing the same item can be installed side-by-side, unless at least one of them uses a `conflicts` array.

I'll still ask the maintainer to add `provides=(ninjamsrv)`. (I used `ninjamsrv` everywhere instead of `ninjam-server` because that's how the binary is called by default and didn't want to confuse users)

Reply:
Thinking some more about this, I do think that naming the package `ninjam-server` would actually be *less* confusing:

1. The binary archive downloadable from cockos.com is called `ninjam_server_*.zip`.
2. The link to that archive is labeled "NINJAM server [...]".
3. The directory in the sources is called `ninjam/server`.
4. The name spells out what the package is for more clearly than the abbreviation `ninjamsrv`, IMO.

Maybe we should get some additional opinions on this.